vllm/compilation/compiler_interface.py

```diff
@@ -133,6 +139,9 @@ class InductorAdaptor(CompilerInterface):
     """
     name = "inductor"
 
+    def __init__(self, compilation_config: CompilationConfig):
```

Comment:
I think we can add a new `Torch28InductorAdaptor` to use the standalone api. since this will only be used for pytorch nightly, we can use it by default when we know pytorch version.

Reply:
This makes sense to me. I added a new InductorStandaloneAdaptor that is currently configurable with compilation_config.use_standalone_compile.

I do want to switch this to being toggled by the PyTorch version, but I want to start out with a flag that I can flip so that changes are easy to turn off/on.